packages/manager/modules/log-to-customer/src/live-tail/controller.js

```diff
@@ -25,20 +31,100 @@ export default class LogToCustomerCtrl {
       createAccount: `${this.dbaasLogsURL}/order`,
     };
 
+    this.kind = this.kindInitValue;
     this.hasAtLeastOneDataStream = false;
+    this.errorMessage = null;
 
-    this.kind = this.kindInitValue;
+    this.init();
+  }
 
-    this.LogToCustomer.getDataStreams()
-      .then(({ streams, streamCount }) => {
-        this.logAccountStreams = streams;
-        this.hasAtLeastOneDataStream = streamCount > 0;
+  init() {
+    this.resetErrorMessage();
+    this.loading = true;
+    this.streamSubscriptions = [];
+
+    this.$q
+      .all([this.getDataStreams(), this.getSubscribedStreams()])
+      .catch(({ data }) => {
+        this.errorMessage = data.message;
       })
       .finally(() => {
         this.loading = false;
       });
   }
 
+  resetErrorMessage() {
+    this.errorMessage = null;
+  }
+
+  deleteLogSubscription(subscription) {
+    this.resetErrorMessage();
+    // eslint-disable-next-line no-param-reassign
+    subscription.isLoading = true;
```

Comment:
Several subscriptions can be managed at the same time ?

Reply:
yes